ui/components/component-library/avatar-base/README.mdx

```diff
@@ -0,0 +1,99 @@
+import { Story, Canvas, ArgsTable } from '@storybook/addon-docs';
+
+import { AvatarBase } from './avatar-base';
+
+### This is a base component. It should not be used in your feature code directly but as a "base" for other UI components
+
+# AvatarBase
+
+The `AvatarBase` is a wrapper component responsible for enforcing dimensions and border radius for Avatar components
+
+<Canvas>
+  <Story id="ui-components-component-library-avatar-base-avatar-base-stories-js--default-story" />
+</Canvas>
+
+## Props
+
+The `AvatarBase` accepts all props below as well as all [Box](/docs/ui-components-ui-box-box-stories-js--default-story#props) component props.
+
+<ArgsTable of={AvatarBase} />
+
+### Size
+
+Use the `size` prop to set the size of the `AvatarBase`.
+
+Possible sizes include:
+
+- `xs` 16px
+- `sm` 24px
+- `md` 32px
+- `lg` 40px
+- `xl` 48px
+
+Defaults to `md`
+
+<Canvas>
+  <Story id="ui-components-component-library-avatar-base-avatar-base-stories-js--size" />
+</Canvas>
+
+```jsx
+import { SIZES } from '../../../helpers/constants/design-system';
+import { AvatarBase } from '../ui/component-library/avatar-base/avatar-base/avatar-base';
+  <AvatarBase {...args} size={SIZES.XS} />
+  <AvatarBase {...args} size={SIZES.SM} />
+  <AvatarBase {...args} size={SIZES.MD} />
+  <AvatarBase {...args} size={SIZES.LG} />
+  <AvatarBase {...args} size={SIZES.XL} />
+```
+
+### Children
+
+The `AvatarBase` component can contain images, icons or text
+
+<Canvas>
+  <Story id="ui-components-component-library-avatar-base-avatar-base-stories-js--children" />
+</Canvas>
+
+```jsx
+import { SIZES } from '../../../helpers/constants/design-system';
+  <AvatarBase {...args}>
+    <img src="./images/eth_logo.svg" />
+  </AvatarBase>
+  <AvatarBase {...args}>
+    <img width="100%" src="./images/arbitrum.svg" />
+  </AvatarBase>
+  <AvatarBase {...args}>
+    <img width="100%" src="./images/avax-token.png" />
+  </AvatarBase>
+  <AvatarBase {...args}>A</AvatarBase>
+```
+
+### Color, Background Color And Border Color
+
+Use the `color`, `backgroundColor` and `borderColor` props to set the text color, background-color and border-color of the s of the `AvatarBase`.
+
+<Canvas>
+  <Story id="ui-components-component-library-avatar-base-avatar-base-stories-js--color-background-color-and-border-color" />
+</Canvas>
+
+````jsx
+import { COLORS, BACKGROUND_COLORS, BORDER_COLORS } from '../../../helpers/constants/design-system';
+import { AvatarBase } from '../ui/component-library/avatar-base/avatar-base/avatar-base';
+  <AvatarBase {...args}>B</AvatarBase>
+  <AvatarBase
+    {...args}
+    backgroundColor={BACKGROUND_COLORS.GOERLI}
+    borderColor={BORDER_COLORS.GOERLI}
+    color={COLORS.PRIMARY_INVERSE}
+    >
+    G
+  </AvatarBase>
+  <AvatarBase
+    {...args}
+    backgroundColor={BACKGROUND_COLORS.SEPOLIA}
+    borderColor={BORDER_COLORS.SEPOLIA}
+    color={COLORS.PRIMARY_INVERSE}
+  >
+    S
+  </AvatarBase>
```

Comment:
Fix imports(mb!) remove storybook specific code. We can just suggest `COLORS` for all color props as per house keeping checklist

```suggestion
import { COLORS } from '../../../helpers/constants/design-system';
import { AvatarBase } from '../../component-library';

<AvatarBase>B</AvatarBase>
<AvatarBase
  backgroundColor={COLORS.GOERLI}
  borderColor={COLORS.GOERLI}
  color={COLORS.PRIMARY_INVERSE}
  >
  G
</AvatarBase>
<AvatarBase
  backgroundColor={COLORS.SEPOLIA}
  borderColor={COLORS.SEPOLIA}
  color={COLORS.PRIMARY_INVERSE}
>
  S
</AvatarBase>
```